ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1941,6 +1941,37 @@ def test_tensorflow_rsqrt(
     )
 
 
+# segment_mean
+@handle_frontend_test(
+    fn_tree="tensorflow.math.segment_mean",
+    data=helpers.array_values(dtype=ivy.int32, shape=(5, 6), min_value=1, max_value=9),
+    segment_ids=helpers.array_values(
+        dtype="int32", shape=(5,), min_value=0, max_value=4
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_segment_mean(
+    *,
+    data,
+    segment_ids,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+    on_device,
+):
+    helpers.test_frontend_function(
+        input_dtypes=["int32", "int64"],
```

Comment:
Can we not generate those using dtype_and_x in the handle_frontend_test decrator like we do for the other tests?